application/Modules/Quotes/Models/QuoteItem.php

```
@@ -134,7 +161,7 @@ public function delete($item_id): bool
         $this->db->where('item_id', $item_id);
         $this->db->delete('ip_quote_item_amounts');
 
-        $this->load->model('quotes/mdl_quote_amounts');
+        $this->load->model('quotes/quote_amount');
         $global_discount['item'] = $this->mdl_quote_amounts->get_global_discount($quote_id);
```

PHPMD (2.15.0)

Avoid using undefined variables such as '$global_discount' which will lead to PHP notices. (undefined)

(UndefinedVariable)